server/src/service/PurchaseOrderService.ts

```diff
@@ -0,0 +1,53 @@
+import { PurchaseOrder } from "../models/store/purchaseOrder";
+import { PurchaseOrderItemInput } from "../models/store/purchaseOrderItemInput";
+import { IPurchaseOrderRepo } from "../repositories/Store/poRepository";
+
+export class PurchaseOrderService {
+  poRepo: IPurchaseOrderRepo;
+
+  constructor(repo: IPurchaseOrderRepo) {
+    this.poRepo = repo;
+  }
+
+  public async getAllPurchaseOrders(): Promise<PurchaseOrder[]> {
```

Comment:
If all the service class does is call the repo, do we need it? Seems like a [middle man](https://refactoring.guru/smells/middle-man) to me, at least at first glance.

Reply:
Don't reply to this thread. Redirecting convo to https://github.com/rit-construct-makerspace/makerspace/pull/82#discussion_r798776165